src/main/java/uk/gov/hmcts/reform/civil/utils/HearingUtils.java

```diff
@@ -54,4 +59,55 @@ public static BigDecimal getFastTrackFee(int claimFee) {
             return new BigDecimal(34600);
         }
     }
+
+    public static String getHearingType(CaseData caseData) {
+        switch (caseData.getChannel()) {
+            case IN_PERSON:
+                return caseData.getHearingLocation().getValue().getLabel();
+            case VIDEO:
+                return "videoconference";
+            case TELEPHONE:
+                return "telephone";
+            default:
+                return "not defined";
+        }
+    }
+
+    public static String getHearingDuration(CaseData caseData) {
+        switch (caseData.getHearingDuration()) {
+            case MINUTES_30:
+                return "30 minutes";
+            case MINUTES_60:
+                return "1 hour";
+            case MINUTES_90:
+                return "1 and half hours";
+            case MINUTES_120:
+                return "2 hours";
+            case MINUTES_150:
+                return "2 and half hours";
+            case MINUTES_180:
+                return "3 hours";
+            case MINUTES_240:
+                return "4 hours";
+            case DAY_1:
+                return "1 day";
+            case DAY_2:
+                return "2 days";
+            default:
+                return "not defined";
```

Comment:
A question on this, since we're generating a template document.

Are we dealing only with english or do we expect to deal also with Welsh?

Reply:
At the moment, it is only in English